src/app/utils.py

```diff
@@ -432,17 +581,18 @@ def get_license_data(issues):
     licenseIds = []
     licenseTexts = []
     for issue in issues:
-        if issue.get('pull_request') is None:
-            licenseInfo = issue.get('body')
-            if '[SPDX-Online-Tools]' in issue.get('title'):
-                licenseIdentifier = re.search(r'(?im)short identifier:\s([a-zA-Z0-9|.|-]+)', licenseInfo).group(1)
-                licenseIds.append(licenseIdentifier)
-                try:
-                    licenseXml = str(LicenseRequest.objects.get(shortIdentifier=licenseIdentifier).xml)
-                    licenseText = parseXmlString(licenseXml)['text']
-                    licenseTexts.append(clean(licenseText))
-                except LicenseRequest.DoesNotExist:
-                    pass
+        if not isinstance(issue, unicode):
+            if issue.get('pull_request') is None:
+                licenseInfo = issue.get('body')
+                if '[SPDX-Online-Tools]' in issue.get('title'):
```

Comment:
What if `issue.get('title')` returns None? If you are 100% sure there is always a title use instead `issue['title']` otherwise if you are not 100% sure and you always want a string use get with an empty string default: `issue.get('title', '')`

Reply:
This too is @Ugtan 's code, so it will be better if he has a look

Reply:
@pombredanne The line is used to filter issues that are created using the SPDX-Online-Tools only but `issue.get('title', '')` would work here.